test/problems/second_quantization/molecular/integrals_calculators/test_angular_momentum_integrals_calculator.py

```diff
@@ -0,0 +1,92 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests total angular momentum integrals calculator."""
+from test import QiskitNatureTestCase
+import numpy as np
+
+from qiskit_nature.problems.second_quantization.molecular.integrals_calculators import \
+    calc_total_ang_momentum_ints
+
+
+class TestAngularMomentumIntegralsCalculator(QiskitNatureTestCase):
+    """Tests total angular momentum integrals calculator."""
+
+    num_modes_list = [1, 2, 3]
+    expected_h_1_list = [[[0.]], [[0.75, 0.],
+                                  [0., 0.75]], [[0.75, 0., 0.],
+                                                [0., 0.75, 0.],
+                                                [0., 0., 0.]]]
```

Comment:
Please reformat this list and the one below to be more readable. E.g. in the following style:
```suggestion
    # A list of expected h1 integrals. Each entry in the list is the expected
    # result for a different number of `num_modes`.
    expected_h_1_list = [
        [[0.]],
        [[0.75, 0.],
         [0., 0.75]],
        [[0.75, 0., 0.],
         [0., 0.75, 0.],
         [0., 0., 0.]]
    ]
```